DTAdmin/MainTabBarController.swift

```diff
@@ -31,10 +31,12 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
         tabBarViewControllers.append(subjectNavController)
 
 //MARK: Create groups tab
-        let groupsTab = TabThreeViewController()
+        let groupStoryboard = UIStoryboard.stoyboard(by: .group)
+        guard let groupNavController = groupStoryboard.instantiateViewController(withIdentifier: "GroupNavController") as? UINavigationController else { return }
+//        let groupsTab = TabThreeViewController()
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.